cmd/workspace.go

```diff
@@ -36,11 +45,90 @@ nothing will happen.
 		// Ignore error. If the file doesn't exist, that is fine.
 		_ = v.ReadInConfig()
 
-		fmt.Fprintf(Out, "%s\n", v.GetString("workspace"))
+		
+
+		var workspaceDir = v.GetString("workspace")
+		if checkTrack {
+			treeLevel = 1
+			treePrint(workspaceDir)
+		} else if checkExercise {
```

Comment:
I believe checkExercise should be the first case in the if statement  as printing the exercise slugs also prints track names.